test/Performance/HelloWorld.cs

```diff
@@ -0,0 +1,117 @@
+    {
+        private static readonly string NetCoreAppTfm = "netcoreapp1.0";
+        private static readonly string s_expectedOutput = "Hello World!" + Environment.NewLine;
+        private static readonly string s_testdirName = "helloworldtestroot";
+        private static readonly string s_outputdirName = "test space/bin";
+
+        private static string RestoredTestProjectDirectory { get; set; }
+
+        private string Rid { get; set; }
+        private string TestDirectory { get; set; }
+        private string TestProject { get; set; }
+        private string OutputDirectory { get; set; }
+
+        static HelloWorld()
+        {
+            HelloWorld.SetupStaticTestProject();
+        }
+
+        public static void Main()
+        {
+            Console.WriteLine("Dummy Entrypoint.");
+        }
+
+        public HelloWorld()
+        {
+            TestInstanceSetup();
+        }
+
+        [Benchmark]
+        public void MeasureDotNetBuild()
+        {
+            foreach (var iter in Benchmark.Iterations)
+            {
+                // Setup a new instance of the test project.
+                TestInstanceSetup();
+
+                // Setup the build command.
+                var buildCommand = new BuildCommand(TestProject, output: OutputDirectory, framework: NetCoreAppTfm);
+                using (iter.StartMeasurement())
+                {
+                    // Execute the build command.
+                    buildCommand.Execute();
+                }
+            }
+        }
+
+        private void TestInstanceSetup()
+        {
+            var root = Temp.CreateDirectory();
+
+            var testInstanceDir = root.CopyDirectory(RestoredTestProjectDirectory);
+
+            TestDirectory = testInstanceDir.Path;
+            TestProject = Path.Combine(TestDirectory, "project.json");
+            OutputDirectory = Path.Combine(TestDirectory, s_outputdirName);
+
+            Rid = PlatformServices.Default.Runtime.GetLegacyRestoreRuntimeIdentifier();
+        }
+
+        private static void SetupStaticTestProject()
+        {
+            RestoredTestProjectDirectory = Path.Combine(AppContext.BaseDirectory, "bin", s_testdirName);
+
+            // Ignore Delete Failure
+            try
+            {
+                Directory.Delete(RestoredTestProjectDirectory, true);
+            }
+            catch (Exception) { }
+
+            Directory.CreateDirectory(RestoredTestProjectDirectory);
+            WriteNuGetConfig(RestoredTestProjectDirectory);
+
+            var currentDirectory = Directory.GetCurrentDirectory();
+            Directory.SetCurrentDirectory(RestoredTestProjectDirectory);
+
+            new NewCommand().Execute().Should().Pass();
+            new RestoreCommand().Execute("--quiet").Should().Pass();
+
+            Directory.SetCurrentDirectory(currentDirectory);
+        }
+
+        // Todo: this is a hack until corefx is on nuget.org remove this After RC 2 Release
```

Comment:
If the intent is to measure the build time of a `dotnet new` empty project, then I think actually using `dotnet new` to generate the project is best. The reasoning is because as soon as you check something into `TestAssets` it will start diverging from what `dotnet new` lays down.

If the intent is to measure any old `Hello, World!` app, then I agree with @pakrym, we can just use something in `TestAssets`.